featuretools/config.py

```diff
@@ -53,8 +53,8 @@ def initialize_logging(config):
     err_handler.setFormatter(logging.Formatter(fmt))
     err_levels = ['WARNING', 'ERROR', 'CRITICAL']
 
-    for name, level in loggers.items():
-        LEVEL = logging._levelNames[level.upper()]
```

Comment:
this private variable changed in python3, `getattr` should have the same functionality and work across python 2.x and 3.x